tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
@Anupama-Github I did intend that you pass the file object, returned from `open()` to the `EnglishChecker` constructor. But in the test case, you can avoid having a dictionary file, by using an in memory file object, as shown below.

```python
    def get_marys_dictionary():
        english_words = ["had", "a", "little", "lamb"]
        dict_file = io.StringIO("\n".join(english_words))
        return dict_file
    ...

    class EnglishCheckerTestCase(unittest.TestCase):
        def test_for_english(self):
            dict_file = get_marys_dictionary()
            checker = utils.EnglishChecker(dict_file)
```
For more details about StringIO see https://docs.python.org/3/library/io.html